pyta.py

```diff
@@ -14,11 +14,17 @@
 
 import webbrowser
 import os
+import sys
 from urllib.request import pathname2url
 
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
+# check the python version
+if sys.version_info > (3,4,0):
+    sys.stderr.write('You need Python 3.4 or later to run this script\n')
```

Comment:
Don't use stderr (print is fine), and the version check is not actually for running PyTA, but for the course itself.